torch/_tensor_str.py

```diff
@@ -285,7 +302,7 @@ def _str(self):
             else:
                 tensor_str = _tensor_str(self, indent)
 
-    if self.layout != torch.strided:
+    if not self.is_quantized and self.layout != torch.strided:
```

Comment:
why do you need this? is quantized a layout?

Reply:
layout was not registered, I'll add it.